tests/.pylintrc

```diff
@@ -9,5 +9,6 @@ variable-rgx=_?_?(?:(?:[a-z0-9]+(?:_[a-z0-9]+)*_?_?)|(?:[a-z0-9]+(?:[A-Z][a-z0-9
 
 [MESSAGES CONTROL]
 disable=
+    duplicate-code,
```

Comment:
Is it because of the instantiation of builder apps?
I think we should not disable this warning even for tests (although we did in `qiwis`).
Because we can define non-testing methods whose names do not begin with "test_".
Just as you did for `get_thread()`.

Reply:
I added this to ignore the PyQt warning in terms of the following code.
```python
def setUp(self):
        self.qapp = QApplication([])
        patcher = mock.patch("requests.get")
        self.mocked_get = patcher.start()
        self.mocked_response = self.mocked_get.return_value
        self.addCleanup(patcher.stop)
```

This method is located in both `test_explorer.py` and `test_builder.py`, thus it makes the warning.
What solution is better?

Reply:
Oh, that was the problem.
Then how about adding pylint disable comments instead?